.composer-require-checker.json

```diff
@@ -1,5 +1,7 @@
 {
```

Comment:
> @keradus I have only doubt about PHAR release - should we include these packages there, or it will work in PHAR too if packages were installed in the project which uses PHAR? 🤔

I am not sure for 2nd part of the question, please feel free to check it out.
but for first part - yes, for phar, we should embed all suggested dependencies

Reply:
Can you point me where's the PHAR build process and where I should hook into, to add suggested deps before build? If we bundle them, there's no need to check if these deps are installed locally 😉.

Reply:
> Can you point me where's the PHAR build process and where I should hook into, to add suggested deps before build? If we bundle them, there's no need to check if these deps are installed locally 😉.

that's valid only for phar distribution. one can still install source version